modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -148,7 +150,7 @@ JPMS introduces stricter encapsulation, impacting how Mule modules interact with
 
 === Add Missing Code
 
-Because reflection is more restricted in Java 17, API objects now require setters. Previously, API objects and plain old Java objects (POJOs) had default constructors and getters for all properties. Now, API objects and POJOs must also have setters so DataWeave can build outside the connector in the Mule app. 
+Because reflection is more restricted in Java 17, API objects now require setters. Previously, API objects and plain old Java objects (POJOs) had default constructors and getters for all properties. Now, API objects and POJOs must also have setters so DataWeave can build outside the connector in the Mule app.
```

Comment:
>  Previously, API objects and plain old Java objects (POJOs) had default constructors and getters for all properties

With Java 8, it is enough for API Java objects to simply offer a default constructor and getters for all DataWeave accessible properties. With Java 17,  they must also have setters so DataWeave can write those properties without using reflection.